app/controllers/admin/tags_controller.rb

```diff
@@ -6,7 +6,6 @@ class TagsController < Admin::ApplicationController
       id supported rules_markdown short_summary pretty_name bg_color_hex
       text_color_hex user_id alias_for badge_id requires_approval
       social_preview_template wiki_body_markdown submission_template
-      name
```

Comment:
This being on its own line felt to me like it might've been intentional, so I checked where it came from and found [this comment](https://github.com/forem/forem/pull/14049/files#r656543551). @juliannatetreault do you know if that comment is still true?

Would it make sense to have separate `create` and `update` params?

Reply:
I'm in the process of testing this now, @jgaskins 😊

Reply:
smart! good catch

Reply:
Just finished testing this out locally and it doesn't appear to be an issue any longer, as I was able to create and update tags without the `name` param (and none of the buttons were disabled like they were at the time that I opened that PR), so this should be good to go! 🚀 

cc @jgaskins and @djuber

Reply:
I'm just now seeing @djuber's comment... what Dan said! 😂

Reply:
Ballin'. Thank you both for checking! 💯